google/cloud/aiplatform/metadata/metadata.py

```diff
@@ -74,6 +101,11 @@ def start_run(self, run: str):
             schema_title=constants.SYSTEM_METRICS,
             schema_version=constants.SCHEMA_VERSIONS[constants.SYSTEM_METRICS],
         )
+        if metrics_artifact.schema_title != constants.SYSTEM_METRICS:
+            raise ValueError(
```

Comment:
do you need something similar in the log_metrics and log_params methods?

Reply:
As we are not only creating/setting metrics artifacts in the `start_run` method, I don't think we would need similar checked for the log_xxx methods. The idea is that the start_run would make sure the run and metrics resource would be setup correctly, and for the log_xxx methods, it would just need to make sure the resources are set and do not need to worries about whether the resources are valid or not.